ccm_web/server/test/app.e2e-spec.ts

```diff
@@ -12,25 +13,17 @@ describe('APIController (e2e)', () => {
     }).compile()
 
     app = moduleFixture.createNestApplication()
+    app.use(cookieParser('SOME_COOKIE_SECRET'))
```

Comment:
Why this specific value, `'SOME_COOKIE_SECRET'`?  Should this be gotten from the application configuration?

Reply:
I mean, this is a test, so I don't think it really needs the actual configuration value. I think I'll probably re-work some of this eventually, but this makes it work for now, since the test module has access to the cookie properties off of `express.Request`.

Reply:
I saw in some documentation that `cookieParser()` is called without arguments.  Does that mean a value is optional here?  If so, would no secret make the cookies less secure?

Reply:
I believe so? I'm not sure I can omit the secret in the end-to-end test, because the extractor function in `jwt.strategy.ts` checks `signedCookies`, not `cookies...